scripts/build_podspec.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
```

Comment:
I'm fairly sure the sanity checking script will complain about the newline between the shebang and the license header. You can run it on your machine with `scripts/sanity.sh` and it should not output errors.

Reply:
Done!